src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
Yup @daffl is right. We'll need to do the same thing with `patch`. Let's merge this, I'll update the `patch` method right now and do a patch release.